shared/pages/Swap/SwapList/SwapList.js

```diff
@@ -51,87 +79,71 @@ export default class SwapList extends Component {
+          this.state.flow.step >= 2 || (this.state.flow.step > 2 && this.state.flow.step <= 8) ? (
+            <div style={{ paddingTop: isMobile ? '50px' : '' }} styleName={this.state.flow.step >= 2 && this.state.flow.step < 5  ? 'stepItem active' : 'stepItem active checked'}>
+              <span styleName="stepNumber">{this.state.flow.step >= 2 && this.state.flow.step < 5 ? '2' : <i className="fas fa-check" />}</span>
               <p styleName="stepText">
-                <FormattedMessage
-                  id="BtcToEthToken58"
-                  defaultMessage="Bitcoin deposition" />
+                <FormattedMessage id="swapList97" defaultMessage="{Currency} deposition" values={{ Currency: sellCurrency === 'BTC' ? sellCurrency : buyCurrency }} />
               </p>
             </div>
           ) : (
             <div styleName="stepItem">
               <span styleName="stepNumber">{2}</span>
               <p styleName="stepText">
-                <FormattedMessage
-                  id="BtcToEthToken67"
-                  defaultMessage="Bitcoin deposition" />
+                <FormattedMessage id="swapList97" defaultMessage="{Currency} deposition" values={{ Currency: sellCurrency === 'BTC' ? sellCurrency : buyCurrency }} />
               </p>
             </div>
           )
         }
 
         {
-          this.props.data.step >= 5 ? (
-            <div style={{ paddingTop: isMobile ? '100px' : '' }} styleName={this.props.data.step >= 5 && this.props.data.step < 6 ? 'stepItem active' : 'stepItem active checked'}>
-              <span styleName="stepNumber">{this.props.data.step >= 5 && this.props.data.step < 6 ? '3' : <i className="fas fa-check" />}</span>
+          this.state.flow.step >= 5 ? (
+            <div style={{ paddingTop: isMobile ? '100px' : '' }} styleName={this.state.flow.step === 5 ? 'stepItem active' : 'stepItem active checked'}>
+              <span styleName="stepNumber">{this.state.flow.step === 5 ? '3' : <i className="fas fa-check" />}</span>
               <p styleName="stepText">
-                <FormattedMessage
-                  id="BtcToEthToken80"
-                  defaultMessage="Swap tokens deposition" />
+                <FormattedMessage id="swapList97" defaultMessage="{Currency} deposition" values={{ Currency: sellCurrency === 'BTC' ? buyCurrency : sellCurrency }} />
               </p>
             </div>
           ) : (
             <div styleName="stepItem">
               <span styleName="stepNumber">{3}</span>
               <p styleName="stepText">
-                <FormattedMessage
-                  id="BtcToEthToken89"
-                  defaultMessage="Swap tokens deposition" />
+                <FormattedMessage id="swapList97" defaultMessage="{Currency} deposition" values={{ Currency: sellCurrency === 'BTC' ? buyCurrency : sellCurrency }} />
```

Comment:
```suggestion
                <FormattedMessage id="swapList97" defaultMessage="{currency} deposition" values={{ currency: sellCurrency === 'BTC' ? buyCurrency : sellCurrency }} />
```
 
переменные лучше называть с маленькой буквы